model/patch_lifecycle.go

```diff
@@ -28,27 +28,39 @@ import (
 // VariantTasksToTVPairs takes a set of variants and tasks (from both the old and new
 // request formats) and builds a universal set of pairs
 // that can be used to expand the dependency tree.
-func VariantTasksToTVPairs(in []patch.VariantTasks) []TVPair {
-	out := []TVPair{}
+func VariantTasksToTVPairs(in []patch.VariantTasks) ([]TVPair, []TVPair) {
```

Comment:
do we want to define a struct type with two named fields for display/execution TVPair slices, to avoid confusion about which comes first? 

which might let the function on line 51 become a method, etc.

Reply:
made it into a single struct